source/slang/slang-ast-stmt.h

```diff
@@ -79,6 +79,13 @@ class IfStmt : public Stmt
     Stmt* negativeStatement = nullptr;
 };
 
+class IfLetStmt: public IfStmt
```

Comment:
I am not sure if we want to inherit from IfStmt as they might be different enough.

Reply:
Right, in this case we are desiguring a single IfLet into a DeclStmt + IfStmt. There is no inheritance relationship between the two.

Reply:
done